cmd/tailscale/tailscale.go

```diff
@@ -108,41 +98,22 @@ func main() {
 					fmt.Fprintf(os.Stderr, "\nTo authorize your machine, visit (as admin):\n\n\t%s/admin/machines\n\n", *server)
 				case ipn.Starting, ipn.Running:
 					// Done full authentication process
+					fmt.Fprintf(os.Stderr, "\ntailscaled is authenticated, nothing more to do.\n\n")
```

Comment:
will this be spammy? If this is going to say this every time, at least trim it down to "Tailscaled authenticated." ?

Reply:
It'll print this once per run of the `tailscale` CLI, because it exits right after printing this. In this current (to be replaced) CLI paradigm, you run `tailscale` and it tells you one of "needs auth, go to this URL" or "no auth needed, all's well".

It's getting replaced with a new CLI soon.